WalletWasabi/EventSourcing/ArenaDomain/CommandProcessor/RoundCommandProcessor.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WalletWasabi.EventSourcing.ArenaDomain.Aggregates;
+using WalletWasabi.EventSourcing.ArenaDomain.Command;
+using WalletWasabi.EventSourcing.ArenaDomain.Events;
+using WalletWasabi.EventSourcing.Interfaces;
+using WalletWasabi.WabiSabi.Backend.Rounds;
+
+namespace WalletWasabi.EventSourcing.ArenaDomain.CommandProcessor
+{
+	public class RoundCommandProcessor : ICommandProcessor
+	{
+		public IEnumerable<IEvent> Process(StartRoundCommand command, RoundState2 aggregate)
+		{
```

Comment:
validate state.Phase == Phase.New (create 'New' phase which will be the default in RoundState2)

Reply:
or I guess you can just check that state.RoundParameters is null that's the same idea